projects/packages/publicize/src/js/classic-editor-connections.js

```diff
@@ -20,6 +20,23 @@ jQuery( function ( $ ) {
 		timer = setTimeout( publicizeConnTestStart, 2000 );
 	} );
 
+	const countConnectionsBy = ( status, response ) => {
+		return ! response.data
+			? 0
+			: response.data.reduce( ( count, testResult ) => {
+					if (
+						! testResult.connectionTestPassed &&
+						status === testResult.connectionTestErrorCode
```

Comment:
...we'd need to make a similar change here, maybe something like 
```suggestion
						status === testResult.connectionTestErrorCode ?? 'broken'
```